src/Security/Authentication/JwtBearer/src/JwtBearerOptions.cs

```diff
@@ -111,5 +111,15 @@ public new JwtBearerEvents Events
         /// from returning an error and an error_description in the WWW-Authenticate header.
         /// </summary>
         public bool IncludeErrorDetails { get; set; } = true;
+
+        /// <summary>
+        /// 1 day is the default time interval that afterwards, <see cref="ConfigurationManager" /> will obtain new configuration.
+        /// </summary>
+        public TimeSpan AutomaticRefreshInterval { get; set; } = ConfigurationManager<OpenIdConnectConfiguration>.DefaultAutomaticRefreshInterval;
+
+        /// <summary>
+        /// 30 seconds is the default time interval that must pass for <see cref="ConfigurationManager" /> to obtain a new configuration.
+        /// </summary>
+        public TimeSpan RefreshInterval { get; set; } = ConfigurationManager<OpenIdConnectConfiguration>.DefaultRefreshInterval;
```

Comment:
```suggestion
        public TimeSpan MinimumRefreshInterval { get; set; } = ConfigurationManager<OpenIdConnectConfiguration>.DefaultRefreshInterval;
```

Reply:
Not sure this makes sense as we are settings `RefreshInterval` not `MinimumRefreshInterval` below